src/FSharp.Core/option.fs

```diff
@@ -148,7 +148,7 @@ module Option =
     [<CompiledName("OfNullable")>]
     let inline ofNullable (value: System.Nullable<'T>) =
         if value.HasValue then
-            Some value.Value
```

Comment:
Hah would've assumed the opposite judging by the members' names. 
You'd do a good thing adding a comment here.

Reply:
It's fine as is.